common/locales/en/groups.json

```diff
@@ -96,5 +96,7 @@
     "abuseReported": "Thank you for reporting this violation. The moderators have been notified.",
     "abuseAlreadyReported": "You have already reported this message.",
     "needsText": "Please type a message.",
-    "needsTextPlaceholder": "Type your message here."
+    "needsTextPlaceholder": "Type your message here.",
+    "copyMessageAsToDo": "copy message as to-do",
```

Comment:
Please leave the option on for system messages unless you can think of a
reason why it is bad rather than just (currently) unnecessary. Others might
have a use for it.
On 26 Apr 2015 04:55, "Blade Barringer" <notifications@github.com> wrote:

> In common/locales/en/groups.json
> <https://github.com/HabitRPG/habitrpg/pull/5087#discussion_r29103103>:
>
> > @@ -96,5 +96,7 @@
> >      "abuseReported": "Thank you for reporting this violation. The moderators have been notified.",
> >      "abuseAlreadyReported": "You have already reported this message.",
> >      "needsText": "Please type a message.",
> > -    "needsTextPlaceholder": "Type your message here."
> > +    "needsTextPlaceholder": "Type your message here.",
> > +    "copyMessageAsToDo": "copy message as to-do",
>
> Should this be capitalized?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/5087/files#r29103103>.
>